.ci/build-n-run.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+uname -r
+#build
+cd examples
```

Comment:
You don't have to run `cd`. Instead, use `make -C examples`.

Reply:
You have to check the return value of `make` respectively, otherwise CI has no further actions.